packages/jest-repl/src/cli/runtime-cli.ts

```diff
@@ -118,7 +118,10 @@ export async function run(
       if (esm) {
         await runtime.unstable_importModule(path);
```

Comment:
TLA is node 14 only, while we support node 12. Should we also support an exported function for ESM?

Reply:
Ah.. Sure. Will add it too.